From 0eb30009604ca2e437f0414be59aaaf586d8caa3 Mon Sep 17 00:00:00 2001
From: Tim Schumacher <timschumi@gmx.de>
Date: Mon, 11 Mar 2019 15:12:51 +0100
Subject: [PATCH 4/5] Revert "lineage-sdk: Add ACCELEROMETER_ROTATION_ANGLES to
 system settings"

This reverts commit 2f9375f4bce76fa98b129529bf188af91dca5eb2.

Change-Id: I88392e75170302fda06900abfd222845371d6771
---
 .../lineageos/providers/LineageSettings.java  | 20 -------------------
 1 file changed, 20 deletions(-)

diff --git a/sdk/src/java/lineageos/providers/LineageSettings.java b/sdk/src/java/lineageos/providers/LineageSettings.java
index 9a074cf..9d12102 100644
--- a/sdk/src/java/lineageos/providers/LineageSettings.java
+++ b/sdk/src/java/lineageos/providers/LineageSettings.java
@@ -2013,23 +2013,6 @@ public final class LineageSettings {
                     }
                 };
 
-        /**
-         * Control the type of rotation which can be performed using the accelerometer
-         * if ACCELEROMETER_ROTATION is enabled.
-         * Value is a bitwise combination of
-         * 1 = 0 degrees (portrait)
-         * 2 = 90 degrees (left)
-         * 4 = 180 degrees (inverted portrait)
-         * 8 = 270 degrees (right)
-         * Setting to 0 is effectively orientation lock
-         * @hide
-         */
-        public static final String ACCELEROMETER_ROTATION_ANGLES = "accelerometer_rotation_angles";
-
-        /** @hide */
-        public static final Validator ACCELEROMETER_ROTATION_ANGLES_VALIDATOR =
-                sNonNegativeIntegerValidator;
-
         /**
          * List of long-screen apps.
          */
@@ -2154,7 +2137,6 @@ public final class LineageSettings {
                 LineageSettings.System.HEADSET_CONNECT_PLAYER,
                 LineageSettings.System.ZEN_ALLOW_LIGHTS,
                 LineageSettings.System.TOUCHSCREEN_GESTURE_HAPTIC_FEEDBACK,
-                LineageSettings.System.ACCELEROMETER_ROTATION_ANGLES,
         };
 
         /**
@@ -2330,8 +2312,6 @@ public final class LineageSettings {
                     TOUCHSCREEN_GESTURE_HAPTIC_FEEDBACK_VALIDATOR);
             VALIDATORS.put(DISPLAY_PICTURE_ADJUSTMENT,
                     DISPLAY_PICTURE_ADJUSTMENT_VALIDATOR);
-            VALIDATORS.put(ACCELEROMETER_ROTATION_ANGLES,
-                    ACCELEROMETER_ROTATION_ANGLES_VALIDATOR);
             VALIDATORS.put(LONG_SCREEN_APPS,
                     LONG_SCREEN_APPS_VALIDATOR);
             VALIDATORS.put(FORCE_SHOW_NAVBAR,
-- 
2.17.1

